saq/queue/base.py

```diff
@@ -40,6 +40,12 @@
 logger = logging.getLogger("saq")
 
 
+DEFAULT_SWEPT_JOB_ERROR = "swept"
```

Comment:
this seems passed on in too many places, can it be reduced?

Reply:
Reduced the places, let me know.